sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatThreadAsyncClient.java

```diff
@@ -98,30 +96,16 @@ public void sendChatReadReceipt(String chatThreadId, SendReadReceiptRequest send
      * @throws ErrorException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      */
-    public void sendChatMessage(String chatThreadId, SendChatMessageRequest sendChatMessageRequest, final Callback<String> callback) {
```

Comment:
Please keep this, we made this change to return String instead of SendChatMessageResult